brave/brave-core
# Adds options for connected state

Resolves https://github.com/brave/brave-browser/issues/5772

UI PR: https://github.com/brave/brave-ui/pull/530

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- connect to uphold with non KYC account
- make sure that rewards and panel page gives you dropdown menu if you click on verify wallet button

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
